src/vos/storage_estimator/common/dfs_sb.py

```diff
@@ -252,11 +252,12 @@ def __init__(self):
     def __del__(self):
         self._lib.d_free_string(ctypes.byref(self._data))
 
-    def get_vos_size_str(self, alloc_overhead):
+    def get_vos_size_str(self, alloc_overhead, vospath):
```

Comment:
(pylint-missing-function-docstring) Missing function or method docstring

Reply:
This one is easy to fix.  Just add a documentation string for the function as the first line.


"""This function ..."""

Reply:
Thx! Fixed.